docs/en/stack/security/authentication/overview.asciidoc

```diff
@@ -9,17 +9,22 @@ means (typically referred to as authentication tokens).
 The {stack} authenticates users by identifying the users behind the requests
 that hit the cluster and verifying that they are who they claim to be. The
 authentication process is handled by one or more authentication services called
-<<realms,_realms_>>. 
+<<realms,_realms_>>.
 
 You can use the native support for managing and authenticating users, or
 integrate with external user management systems such as LDAP and Active
-Directory. 
+Directory.
 
-{security} provides built-in realms such as `native`,`ldap`, `active_directory`, 
-`pki`, `file`, and `saml`. If none of the built-in realms meet your needs, you 
-can also build your own custom realm and plug it into the {stack}. 
+{security} provides built-in realms such as `native`,`ldap`, `active_directory`,
+`pki`, `file`, and `saml`. If none of the built-in realms meet your needs, you
+can also build your own custom realm and plug it into the {stack}.
 
 When {security-features} are enabled, depending on the realms you've configured,
 you must attach your user credentials to the requests sent to {es}. For example,
-when using realms that support usernames and passwords you can simply attach 
+when using realms that support usernames and passwords you can simply attach
 {wikipedia}/Basic_access_authentication[basic auth] header to the requests.
+
+{security} provides built-in authentication services such as `token-service` and
```

Comment:
{security} provides two services, the token service and the api key service, which allow the exchange of the current authentication for a token/key to be used for authentication without providing other means of authentication.